tensilelite/Tensile/TensileInstructions/Instructions.py

```diff
@@ -1813,11 +1832,19 @@ def setupInstructions(self):
             lgkmcnt = 0
             vmcnt   = 0
             vscnt   = 0
+            dscnt   = 0
+            kmcnt   = 0
+            loadcnt = 0
+            storecnt= 0
             comment = "(Wait all)"
         else:
             lgkmcnt = self.lgkmcnt
             vmcnt   = self.vmcnt
             vscnt   = self.vscnt
+            kmcnt   = self.kmcnt
+            dscnt   = -1 if kmcnt != -1 else self.lgkmcnt
```

Comment:
why set it as -1 if kmcnt != -1?  Should it be different thing with kmcnt?

Reply:
due to lgkmcnt split into dscnt and kmcnt.
current logic is if kmcnt = -1, dscnt = lgkmcnt, or otherwise dscnt = -1. just set kmcnt or let kmcnt = lgkmcnt